### PR TITLE
Glue.segments is no more

### PR DESCRIPTION
**Pull request by spxiwh**
`glue.segments` has been marked as deprecated for some time now. The latest glue release now removes this in favour of `ligo.segments` (which it had been calling internally for some time anyway). Here we just replace all `glue.segments` with `ligo.segments` and add the corresponding package to the requirements.txt.

A similar change may be needed for `glue.ligolw` at some point in the future as well. (Please let's not discuss forking glue again, that didn't go well the first time).

### CODE REVIEW

**Comment by ahnitz** [comment]
@spxiwh Looking through this looks like the right changes.

File: pycbc/workflow/grb_utils.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 import os
 import shutil
 import urlparse, urllib
-from glue import segments
+from lal import segments
```
Comment (titodalcanton) [inline pycbc/workflow/grb_utils.py]: Is importing from `lal` a typo here? I just tried it on a PyCBC env from yesterday and it does not work.

Comment (ahnitz) [reply pycbc/workflow/grb_utils.py]: @titodalcanton Was it an old virtualenv? I.E. before we were installing lalsuite with pip?

Comment (ahnitz) [reply pycbc/workflow/grb_utils.py]: If so, it might not have had the ligo-segments package. Otherwise, I don't know....

Comment (titodalcanton) [reply pycbc/workflow/grb_utils.py]: I mean, should the import not read `from ligo import segments` like in the rest of the patch?

Comment (spxiwh) [reply pycbc/workflow/grb_utils.py]: Ah, yes

**Comment by spxiwh** [comment]
Problems at UWM are making Travis fail, will need to wait

File: pycbc/workflow/grb_utils.py (hunk 1 above)
Comment (spxiwh) [reply pycbc/workflow/grb_utils.py]: Fixed

**Comment by Cyberface** [comment]
I think git.ligo is back up so I took the liberty of restarting Mr. Travis

**Comment by spxiwh** [comment]
I think `software.ligo.org` is still down, so Travis is still failing

**Pull request merged**